### PR TITLE
Update m-team FlareSolverr har

### PR DESCRIPTION
**Pull request by 15cm**
Changelog:
- Add domain to cookies which is required since FlareSolverr 3.0.0
- Add failed asserts to check again the response of FlareSolverr

Before the User cookies support mentioned in https://github.com/FlareSolverr/FlareSolverr/issues/650 is implemented in the official repo, the har only works with this docker image: `15cm/flaresolverr:3.0.0` or onward. The docker image is built from https://github.com/15cm/FlareSolverr .

Fix https://github.com/qiandao-today/templates/issues/225

### CODE REVIEW

File: tpls_history.json [hunk 1]
```diff
@@ -2061,11 +2061,11 @@
             "author": "Sinkerine",
             "url": "https://raw.githubusercontent.com/qiandao-today/templates/master/m-team-cookie-flaresolverr.har",
             "update": false,
-            "comments": "cookies_value_tp: Cookies 中 \"tp\" key 的值<br>userid: userdetails.php?id={{userid}} 的用户 id<br>FlareSolverr url. 如 http://localhost:8191/v1",
+          "comments": "cookies_value_tp: Cookies 中 \"tp\" key 的值<br>userid: userdetails.php?id={{userid}} 的用户 id<br>FlareSolverr url. 如 http://localhost:8191/v1 . 目前 Docker image 必须用 15cm/flaresolverr:3.0.0 以及更高版本的 DockerHub 镜像，原因见 PR 链接",
```
Comment (a76yyyy) [inline tpls_history.json:2064]: 格式是否对齐

Comment (a76yyyy) [reply tpls_history.json:2064]: 建议说明更简洁清晰一些

File: m-team-cookie-flaresolverr.har [hunk 2]
```diff
@@ -1 +1 @@
-[{"request":{"method":"POST","url":"{{flaresolverr_url}}","headers":[{"name":"Content-Type","value":"application/json"}],"cookies":[],"data":"{\n  \"cmd\": \"request.get\",\n  \"url\":\"https://kp.m-team.cc/userdetails.php?id={{userid}}\",\n  \"cookies\": [{\"name\": \"tp\", \"value\": \"{{cookies_value_tp}}\"}],\n  \"maxTimeout\": 60000\n}"},"rule":{"success_asserts":[{"re":"200","from":"status"}],"failed_asserts":[],"extract_variables":[]}}]
\ No newline at end of file
+[{"request":{"method":"POST","url":"{{flaresolverr_url}}","headers":[{"name":"Content-Type","value":"application/json"}],"cookies":[],"data":"{\n  \"cmd\": \"request.get\",\n  \"url\":\"https://kp.m-team.cc/userdetails.php?id={{userid}}\",\n  \"cookies\": [{\"name\": \"tp\", \"value\": \"{{cookies_value_tp}}\", \"domain\": \"kp.m-team.cc\"}],\n  \"maxTimeout\": 60000\n}"},"rule":{"success_asserts":[{"re":"200","from":"status"}],"failed_asserts":[{"re":".*https://kp.m-team.cc/login.php\\?returnto=.*","from":"content"},{"re":".*\"status\": \"error\".*","from":"content"}],"extract_variables":[]}}]
```
Comment (a76yyyy) [inline m-team-cookie-flaresolverr.har:1]: 是否需要__log__日志?

File: tpls_history.json (hunk 1 above)
Comment (15cm) [reply tpls_history.json:2064]: Done.

File: m-team-cookie-flaresolverr.har (hunk 2 above)
Comment (15cm) [reply m-team-cookie-flaresolverr.har]: 好注意。加上了长度合适的 __log__

**Pull request merged**